package.json

```diff
@@ -1,95 +1,46 @@
 {
-  "name": "@ember/render-modifiers",
-  "version": "2.0.4",
-  "description": "Implements did-insert / did-update / will-destry modifiers for emberjs/rfcs#415",
-  "keywords": [
-    "ember-addon"
-  ],
-  "repository": "git@github.com:emberjs/ember-render-modifiers.git",
-  "license": "MIT",
-  "author": "Robert Jackson <me@rwjblue>",
-  "main": "index.js",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "private": true,
+  "workspaces": ["packages/*"],
   "scripts": {
-    "build": "ember build --environment=production",
+    "prepare": "yarn workspace @ember/render-modifiers run prepare",
+    "start": "npm-run-all --parallel start:*",
+    "start:addon": "yarn workspace @ember/render-modifiers run start",
+    "start:test-app": "yarn workspace test-app run start",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
-  },
-  "dependencies": {
-    "@embroider/macros": "^1.0.0",
-    "ember-cli-babel": "^7.26.11",
-    "ember-modifier-manager-polyfill": "^1.2.0"
+    "lint:addon": "yarn workspace @ember/render-modifiers run lint",
+    "lint:test-app": "yarn workspace test-app run lint",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:fix:*",
+    "lint:fix:addon": "yarn workspace @ember/render-modifiers run lint:fix",
+    "lint:fix:test-app": "yarn workspace test-app run lint:fix",
+    "test": "npm-run-all --aggregate-output --continue-on-error --parallel \"test:!(watch)\"",
+    "test:watch": "npm-run-all --aggregate-output --continue-on-error --parallel test:watch:*",
+    "test:test-app": "yarn workspace test-app run test",
+    "test:watch:test-app": "yarn workspace test-app run test:watch",
+    "test:watch:addon": "yarn workspace @ember/render-modifiers run start",
+    "release": "release-it"
+  },
+  "volta": {
+    "node": "16.14.2",
```

Comment:
@ctjhoa I think this should stay as 12 as minimum supported version and the same what was used in CI.

Or we could remove `engines.node` from `@ember/render-modifiers` as it's purely frontend code and has nothing to do with Node.js versions anymore (fully delegated to build pipeline)

Reply:
I've removed `engines.node`
https://github.com/emberjs/ember-render-modifiers/pull/66/commits/226ff161bb72a1c2ce072fe5bc720d6500f5a2c0

As this PR introduces a breaking change, go for the latest Node LTS seems fine to me.